### DISCUSSION TITLE
middle mouse button on inventoryClickEvent

### DISCUSSION THREAD

**Discussion by AchyMake** [PaperMC/Paper#11488] [Ideas]
### Is your feature request related to a problem?

not sure if its removed or bugged
but i thought middle mouse button used to be enable for survival players

### Describe the solution you'd like.

middle mouse button on inventory click

### Describe alternatives you've considered.

middle mouse button on inventory click

**Comment by electronicboy** [comment]
I thought this was a thing until Mojang broke it a few versions ago
